Projects/Core/Common/Utilities/DebugLogger.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  DebugLogger.swift
+//  Common
+//
+//  Created by GREEN on 6/14/24.
+//  Copyright © 2024 com.mashup.gabbangzip. All rights reserved.
+//
+
+import Foundation
+
+public let logger = DebugLogger()
+
+public class DebugLogger {
+  private static let fileLogQueue = DispatchQueue(label: "com.mashup.gabbangzip.file_log_queue")
+  
+  public init() { }
```

Comment:
P3:
```suggestion
  public init() {}
```
띄어쓰기가 한 칸 더 들어간 것 같습니다